lib/tasks/oauth.rake

```diff
@@ -0,0 +1,23 @@
+namespace :oauth do
+  desc "Approves Coposition default mobile app for all users"
+  task default_mobile_approve: :environment do
+    puts "Adding access tokens for all users:"
+
+    User.find_each(batch_size: 100) do |user|
+      Doorkeeper::AccessToken.find_or_create_for(
+        default_application,
+        user.id,
+        "public",
+        7200,  # default value from server config settings
+        true)  # not a default value, enabling refresh tokens here
+
```

Comment:
Need to complete the approval here also I think, i.e.
```
developer = Developer.default(mobile: true)
developer.approvals.find_by(user_id: current_resource_owner.id).tap do |approval|
  approval ||= Approval.add_developer(current_resource_owner, developer)
  approval.update(status: "complete")
end
```
Or something along those lines.